src/storage/btree.c

```diff
@@ -15775,6 +15956,10 @@ btree_prepare_bts (THREAD_ENTRY * thread_p, BTREE_SCAN * bts, BTID * btid, INDX_
   /* Set other arguments specific to scan type. */
   bts->bts_other = bts_other;
 
+#if defined(IMPROVE_RANGE_SCAN_IN_BTREE)
+  btree_init_common_prefix_page_info (bts);
```

Comment:
I think the btree_init_common_prefix_page_info function is unnecessary.
```suggestion
  COMMON_PREFIX_PAGE_SIZE_RESET (bts);
  btree_clear_key_value (&bts->clear_common_prefix_key, &bts->common_prefix_key);
  //btree_init_temp_key_value (&bts->clear_common_prefix_key, &bts->common_prefix_key); 
```

Reply:
That makes sense.
But I'll keep it like this.

This is because in other issues, there is a need to perform this content without calling btree_prepare_bts(), and it seems that it can be recognized intuitively by leaving it out as a function.
Even if it is a function call, it will not affect performance.

Thank you.

Reply:
Because I think it affects readability.

Variables inside BTS are initialized directly here.
I think there is no need to differentiate and initialize only the variables inside the btree_init_common_prefix_page_info function.

Reply:
Since the purpose is to express my opinion, I will resolve it.